spec.bs

```diff
@@ -1585,10 +1591,15 @@ the script or wasm, otherwise a [=tuple=] of ([=list=] of [=generated bids=],
     1. Let |timeDelta| be |auctionStartTime| minus |prevWin|'s [=previous win/time=].
     1. Set |timeDelta| to 0 if |timeDelta| is negative, |timeDelta|'s nearest second (rounding down)
       otherwise.
-    1. Let |prevWinIDL| be a new {{PreviousWin}}.
-    1. [=map/Set=] |prevWinIDL|["{{PreviousWin/timeDelta}}"] to |timeDelta|.
-    1. [=map/Set=] |prevWinIDL|["{{PreviousWin/adJSON}}"] to |prevWin|'s [=previous win/ad json=].
-    1. [=list/Append=] |prevWinIDL| to |prevWins|.
+    1. Let |prevWinIDL| be a new {{AuctionAd}} with the following [=struct/items=]:
+      : {{AuctionAd/renderURL}}
+      :: |prevWin|'s [=interest group ad/render url=]

```

Comment:
{{AuctionAd/renderURL}} is a DOMString, but [=interest group ad/render url=] is a URL.
You can use URL serializer to convert it to a string, like [this example](https://github.com/WICG/turtledove/blob/main/spec.bs#L1998)

Reply:
Done.